website/versioned_docs/version-0.62/contributing.md

```diff
@@ -0,0 +1,60 @@
+---
+id: version-0.62-contributing
+title: Contributing to microsoft/react-native-windows
+original_id: contributing
+---
+
+If this is your first time, below are some more detailed steps for getting your branch set up and submitting a PR. You can either do this with the command prompt or with a combination of the command prompt and SourceTree.
```

Comment:
This is another one that seems deletable if we aren't including it anywhere. I think it's a copy of what we have in the RNW wiki.

Reply:
removed, linking to it or similar will be tracked with #149